policykit/policyengine/views.py

```diff
@@ -298,42 +298,51 @@ def editor(request):
         except Policy.DoesNotExist:
             raise Http404("Policy does not exist")
 
-    # which action types to show in the dropdown
-    actions = Utils.get_action_types(community, kinds=[kind])
-
-    # list of autocomplete strings
-    action_types = [a.codename for a in policy.action_types.all()] if policy else None
-    autocompletes = Utils.get_autocompletes(community, action_types=action_types, policy=policy)
-
-    # determine whether the page should be displayed as modal or full page
-    base = DASHBOARD_BASE 
-
-    if request.META.get('HTTP_HX_REQUEST'):
-        base = DASHBOARD_BASE_AJAX
-
-    data = {
-        'user': get_user(request),
-        'type': kind.capitalize(),
-        'operation': operation,
-        'actions': actions.items(),
-        'autocompletes': json.dumps(autocompletes),
-        'base': base
-    }
+    if not policy.policy_template:
```

Comment:
I am trying this locally with a URL like `https://loyal-guinea-instantly.ngrok-free.app/no-/editor/?policy=1` and see that it doesn't render the new UI because there is not template for this policy.

Doesn't this mean that any policies not created in the new UI wont have a policy template and so wont be editable with the new UI?